.github/first_interaction.yml

```
@@ -0,0 +1,29 @@
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  :wave: Welcome, and thank you for your first contribution!
+
+  We ask you to take a moment to read through our Contributing Guide:
+
+  :point_right: https://github.com/nipreps/{{repository}}/blob/main/CONTRIBUTING.md
```

I don't know if we can count on this file existing for every repo, and we have a web page to cover this:

```suggestion
  :point_right: https://www.nipreps.org/community/CONTRIBUTING/
```